ksqldb-engine/src/test/java/io/confluent/ksql/query/QueryBuilderTest.java

```diff
@@ -95,7 +95,7 @@
 import org.mockito.Mockito;
 import org.mockito.junit.MockitoJUnitRunner;
 
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
Because the FF being on is ignored it considered unnecessary but are testing that it is ignored so we want it to be the same so we are stoping the unnecessary stubbing alerts